src/packagedcode/plugin_package.py

```diff
@@ -272,6 +274,32 @@ def process_codebase(self, codebase, strip_root=False, package_only=False, **kwa
                 if TRACE_LICENSE and modified:
                     logger_debug(f'packagedcode: process_codebase: add_referenced_license_matches_from_package: modified: {modified}')
 
+@post_scan_impl
+class PackageSummary(PostScanPlugin):
+    """
+    Summary at the Package Level.
+    """
+    run_order = 11
+    sort_order= 11
+
+    options = [
+        PluggableCommandLineOption(('--package-summary',),
+        is_flag=True, default=False,
+        help='Summarize scans by providing License Clarity Score at the '
+        'Package attribute level.',
+        help_group=POST_SCAN_GROUP)
+    ]
+
+    def is_enabled(self, package_summary, **kwargs):
+        return package_summary
+
+    def process_codebase(self, codebase, package_summary, **kwargs):
+        """
+        Process the codebase.
+        """
+        if not self.is_enabled(package_summary):
+            return
```

Comment:
Any progress here? This is the main part.

Reply:
In the next commit, I will be pushing this part.

Reply:
From https://github.com/nexB/scancode-toolkit/pull/3792/commits/65de49a3bf8b5fe99c547420d1c15e5f159bc481 onwards, working on this part.